src/software/logger/network_sink.cpp

```diff
@@ -9,6 +9,7 @@ NetworkSink::NetworkSink(unsigned int channel, const std::string& interface, int
     log_output.reset(new ThreadedProtoUdpSender<TbotsProto::RobotLog>(
         std::string(ROBOT_MULTICAST_CHANNELS.at(channel)) + "%" + interface,
         ROBOT_LOGS_PORT, true));
+    num_repeats = 0;
```

Comment:
as Cody suggested, it'd be good to have this initialized as the initializer list